projects/1.20/assets/productivebees/productivebees/patchouli_books/guide/zh_cn/entries/blocks/centrifuge.json

```
@@ -0,0 +1,12 @@
+{
+    "name": "离心机",
+    "icon": "productivebees:centrifuge",
+    "category": "productivebees:blocks",
+    "pages": [
+        {
+            "type": "patchouli:spotlight",
+            "item": "productivebees:centrifuge",
+            "text": "将资源蜜脾分离成资源和蜂蜜。可容纳 10 桶可抽取的液体。离心机也能自动将流体输出到旁边的任何流体储存方块。动力离心机需要FE来以更高的速度运行。离心机还可用于将蜜蜂材料瓶中的遗传物质分离成$(l:genetics/transferring)基因$(/l)。"
```

```suggestion
            "text": "将资源蜜脾分离成资源和蜂蜜。可容纳 10 桶可抽取的液体。离心机也能自动将流体输出到旁边的任何流体储存方块。为动力离心机提供 FE，可让其以更高的速度运转。离心机还可将蜜蜂碾膏瓶中的遗传物质分离成$(l:genetics/transferring)基因$(/l)。"
```
蜜蜂碾膏为同步zh_cn中的译文，如采纳其他建议则跟随其更改